wiki/Do_you_really_want_to_ask_peppy/pl.md

```diff
@@ -8,7 +8,7 @@ Większość problemów technicznych można rozwiązać poprzez znalezienie odpo
 
 ## Pomoc ogólna
 
-Dla problemów związanych z kontem, opłatami, korespondencji dotyczącej blokad konta, pytań odnośnie turniejów oraz **wszystkich pozostałych kwestii wymagających wsparcia** wyślij e-maila na adres [accounts@ppy.sh](mailto:accounts@ppy.sh) (pytania dotyczące kont) lub [osu@ppy.sh](mailto:osu@ppy.sh) (pozostałe). Jeśli któryś z członków zespołu pomocy technicznej jest dostępny, możesz użyć małego przycisku ze znakiem zapytania, który pojawi się w prawym dolnym rogu.
+Dla problemów związanych z kontem, opłatami, korespondencji dotyczącej blokad konta, pytań odnośnie turniejów oraz **wszystkich pozostałych spraw** wyślij e-maila na adres [accounts@ppy.sh](mailto:accounts@ppy.sh) (pytania dotyczące kont) lub [osu@ppy.sh](mailto:osu@ppy.sh) (pozostałe). Jeśli któryś z członków zespołu pomocy technicznej jest dostępny, możesz użyć małego przycisku ze znakiem zapytania, który pojawi się w prawym dolnym rogu.
```

Comment:
Ty czekaj, bo z przyzwyczajenia się rozpędziłem i edytując końcówkę nie wziąłem pod uwagę tego, że ten wafel chce to mieć teraz na wiki zamiast na głównej stronie jak do tej pory. Nie ma to jak robienie problemów tam, gdzie ich nie ma.

który pojawi się w prawym dolnym rogu -> który pojawi się w prawym dolnym rogu głównej strony osu!

Ewentualnie jakiś inny pomysł? Chociaż z drugiej strony to nie zdziwię się, jak na nowej stronie w ogóle nie będzie tego znaku zapytania (albo co innego). Ćam awruk.